recipes/cmcrameri/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "cmcrameri" %}
+{% set version = "0.12" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9c7a7533647629ec620128c4adbeb03ffc308e4ffef7dab2af7fb87176977d65
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<36]
+
+requirements:
+  host:
+    - matplotlib
```

Comment:
In general, only `python` and `pip` are needed in the `host` section, so can you please check that? Also, do you need to use `matplotlib` or `matplotlib-base`? The former one is bigger package with the Qt backend, but the latter one should be used if the package doesn't depend on interactive images, but uses `matplotlib` to render images to files.